06_correlation_analysis.ipynb

```diff
@@ -0,0 +1,1984 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    symbol_to_entrez, _ = tu.get_symbol_map()
</pre><p>You could also use the first return value instead of using <code>_</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/pancancer-evaluation/pull/46/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>